Lib/webbrowser.py

```diff
@@ -588,14 +629,26 @@ def open(self, url, new=0, autoraise=True):
             return not rc
 
 
+def _get_supported_browsers():
+    _brow = []
+    for bt in _browsers_types:
+        try:
+            _ = get(bt)
```

Comment:
No need to assign to a variable

Reply:
i have not checked with ... flake8 (?) ... i have to admit

Reply:
CPython doesn’t use these tools.  `get(bt)` on the line is enough!

Reply:
https://github.com/kr-g/cpython/commit/fd0e6a39f196838f5c6a626276e7313eb3d6df6b